cmd/gokeyless/gokeyless.go

```diff
@@ -54,31 +55,32 @@ func init() {
 func main() {
 	flag.Parse()
 
-	s, err := server.NewServerFromFile(certFile, keyFile, caFile, net.JoinHostPort("", port), "")
-	if err != nil {
-		log.Warningf("Could not create server. Running initializeServer to get %s and %s", keyFile, certFile)
-
-		// If an existing CSR (and associated key) exists, don't
-		// proceed/overwrite the key.
-		if _, err := os.Stat(csrFile); !os.IsNotExist(err) {
-			log.Fatalf("an existing CSR was found at %q: remove once a signed certificate has been installed.", csrFile)
```

Comment:
I still want us to support the manual activation path as a backup/fallback if the automatic activation path fails. 

Having this guard prevents the end-user from running `$ gokeyless -manual-activation` again by accident, writing a new CSR to disk, and invalidating the one that might still be "in transit" (read: waiting to be signed + returned).